tests/testthat/test-module_teal.R

```diff
@@ -1588,25 +1588,37 @@ testthat::describe("srv_teal teal_module(s) transformer", {
   })
 
   testthat::it("fails when transformer doesn't return reactive", {
-    testthat::expect_error(
-      testServer(
-        app = srv_teal,
-        args = list(
-          id = "test",
-          data = teal.data::teal_data(iris = iris),
-          modules = modules(
-            module(
-              server = function(id, data) data,
-              transformers = list(
-                teal_transform_module(
-                  ui = function(id) NULL,
-                  server = function(id, data) "whatever"
+    testthat::expect_warning(
```

Comment:
Please also add a test for an `reactive.event` warning

Reply:
Done in 31c53012a!

Also added a test in d885b100d for `srv_teal teal_modules` that tests `testthat::it("are called only after teal_data_module is resolved"`